libcaf_net/test/transport_worker_dispatcher.cpp

```diff
@@ -255,9 +262,8 @@ struct fixture : host_fixture {
   test_write_message(testcase);                                                \
   CAF_CHECK_EQUAL(buf->size(), hello_test.size() + 1u);                        \
   CAF_CHECK_EQUAL(static_cast<byte>(testcase.worker_id), buf->at(0));          \
-  CAF_CHECK_EQUAL(memcmp(buf->data() + 1, hello_test.data(),                   \
-                         hello_test.size()),                                   \
-                  0);                                                          \
+  CAF_CHECK_EQUAL(                                                             \
+    memcmp(buf->data() + 1, hello_test.data(), hello_test.size()), 0);         \
```

Comment:
Can't we simply use the byte-wise comparison of `span`?

```suggestion
    make_span(buf->data() + 1, hello_test.size(), make_span(hello_test)); \
```